### ISSUE TITLE
tp.file.cursor() does not work

### ISSUE THREAD

**Issue by qawatake** [SilentVoid13/Templater#583]
**Plugin informations (please complete the following information):**

-   OS: macOS 12.2.1
-   Templater version: 1.11.3
-   Obsidian version: v0.13.33
-   Templater settings: `Trigger Templater on new file creation` enabled. The others are default.

**Describe the bug**
I create a zettelkasten note by the core zettelkasten plugin with the following template:
```
# <% tp.file.cursor() %>
```

After creating a note, the cursor is on the file name field.

**Expected behavior**
After creating a zettelkasten note, the cursor should be inserted after `# `.

**Screenshots**
[![Image from Gyazo](https://i.gyazo.com/04502fc116bea7df45d38215e6112fd4.gif)](https://gyazo.com/04502fc116bea7df45d38215e6112fd4)

**Additional context**
Updating the version caused this bug. Version 1.9.x did not have this bug.

**Comment by liamcain** [comment]
This looks like the expected behavior to me. The zettlekasten prefixer plugin allows you to also provide a descriptive name for your file. So the filename field should stay focused to allow you to add your note title (e.g. `202203130901 my note`). When you press <kbd>Enter</kbd> your cursor will be at the location of `tp.file.cursor()`.

**Comment by qawatake** [comment]
@liamcain 

Thank you for your reply!
I see. I have checked the changelog and found https://github.com/SilentVoid13/Templater/blob/master/CHANGELOG.md#bug-fixes.

By the way, is it possible to make these options selectable?
I would like to have the cursor immediately go to the text, as it was originally supposed to do.

**Comment by liamcain** [comment]
I don't think that behavior was ever the intention of the cursor jump, so I'd rather not add it back even as an option. In my opinion, it was only a bug. However, if you've gotten used to the behavior and would like to recreate it, I recommend adding the following to the top of your template:

```
<%* app.workspace.activeLeaf.view.editor.focus() -%>
```

That will immediately cause the file to get focused on creation, which will be the same behavior you were seeing with the cursor jump previously.

**Comment by qawatake** [comment]
All right, I will try to get used to the new behavior.
(By the way, using the workaround you gave me, the cursor does not seem to be at the exact position I specified.)
